app/Models/Role.ts

```diff
@@ -0,0 +1,22 @@
+import { DateTime } from 'luxon'
+import { BaseModel, column , belongsTo , BelongsTo } from '@ioc:Adonis/Lucid/Orm'
+
+
+export default class Role extends BaseModel {
+  @column({ isPrimary: true })
+  public id: number
+  
+
+  @column({ isPrimary: true })
+  public user_id: number
+  
+  @column()
+  public role: string
```

Comment:
@theshiv12 

we had already discussed this
we will have 3 column
`user` -> contains list of users
`role` -> contain list of roles
`user_role` -> map user and role based on their PK